include/modem/location.h

```diff
@@ -322,6 +337,25 @@ const char *location_method_str(enum location_method method);
  */
 int location_agps_data_process(const char *buf, size_t buf_len);
 
+/**
+ * @brief Feed in P-GPS data to be processed by library.
+ *
+ * @details If Location library is not receiving P-GPS assistance data directly from nRF Cloud,
+ * it throws event LOCATION_EVT_GNSS_PREDICTION_REQUEST when assistance is needed. Once application
+ * has obtained the assistance data it can be handed over to Location library using this function.
+ *
+ * Note that the data must be formatted similarly to the nRF Cloud P-GPS data, see for example
+ * nrf_cloud_pgps_schema_v1.h.
+ *
+ * @param[in] buf Data received.
+ * @param[in] buf_len Buffer size of data to be processed.
+ *
+ * @return 0 on success, or negative error code on failure.
+ * @retval -EINVAL Given buffer NULL or buffer length zero.
```

Comment:
The same modification could be done for `location_agps_data_process()` description.
```suggestion
 * @retval -EINVAL Given buffer is NULL or buffer length zero.
```

Reply:
yes